ironfish/src/mining/stratum/stratumServer.ts

```diff
@@ -245,6 +269,7 @@ export class StratumServer {
     client.socket.removeAllListeners()
     client.close()
     this.clients.delete(client.id)
+    this.addToConnectionCount(client.socket.remoteAddress ?? "", -1)
```

Comment:
Suggestion related to the above change of splitting these fns out

```suggestion
    if (client.socket.remoteAddress) {
      this.removeFromConnectionCount(client.socket.remoteAddress)
    }
```